Makefile

```diff
@@ -1,3 +1,4 @@
+ALL_VERSIONS = 2.3 2.4 2.5 2.6 2.7 2.8 2.9 2.10 2.11 3.0 3.1 3.2 4.0 4.1 4.2 4.3 5.0
```

Comment:
5.1 is not included ?
Isn't there a way to detect this ? if no, you must update the new version procedure